sdk/resourcemanager/azure-resourcemanager-cosmos/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+## 2.28.0-beta.1 (2023-06-01)
```

Comment:
05-31

Reply:
This comment was fixed in the next version